synfig-core/src/synfig/valuenodes/valuenode_add.cpp

```diff
@@ -247,3 +229,39 @@ ValueNode_Add::get_children_vocab_vfunc() const
 
 	return ret;
 }
+
+LinkableValueNode::InvertibleStatus
+ValueNode_Add::is_invertible(Time t, const ValueBase& target_value, int* link_index) const
```

Comment:
Is it possible to use here `const Time& t`?

Reply:
Yeah, I thought that too, but a lot of other places are like that.
Sure it's possible. I'll do it too.

Reply:
Ah, ok. 
Then it's better to fix it in another PR.

Reply:
Already changed that lol